tests/integration/devfile/cmd_delete_test.go

```diff
@@ -252,4 +252,66 @@ ComponentSettings:
 			})
 		})
 	})
+
+	//Test reused and adapted from the now-removed `cmd_devfile_delete_test.go`
+	When("deleting a component that has resources attached to it and was deployed with Dev", func() {
```

Comment:
Test reused and adapted from the now-removed [cmd_devfile_delete_test.go](https://github.com/redhat-developer/odo/blob/24fd02673d25eb4c7bb166ec3369554a8e64b59c/tests/integration/devfile/cmd_devfile_delete_test.go#L172-L238):

Original version, which leveraged `odo url`:
https://github.com/redhat-developer/odo/blob/24fd02673d25eb4c7bb166ec3369554a8e64b59c/tests/integration/devfile/cmd_devfile_delete_test.go#L172-L238